.ci/test-r-package-valgrind.sh

```diff
@@ -2,7 +2,7 @@
 
 set -e -E -u -o pipefail
 
-RDscriptvalgrind -e "install.packages(c('R6', 'data.table', 'jsonlite', 'Matrix', 'RhpcBLASctl', 'testthat'), repos = 'https://cran.rstudio.com')" || exit 1
+RDscriptvalgrind ./.ci/install-r-deps.R --build --test || exit 1
```

Comment:
Why do we need `--build` here? I don't see "knitr" and "markdown" in `master`, and we pass `--no-build-vignettes` at the next line.

 ```suggestion
RDscriptvalgrind ./.ci/install-r-deps.R --test || exit 1
```